uilib/fileManager.py

```diff
@@ -24,6 +25,10 @@ def __init__(self, app):
         self.fileName = None
 
         self.newFile()
+        
+        self.recentlyOpenedFiles = []
+        self.recentFiles = getConfigPath() + 'recentfiles.txt'
```

Comment:
It is a pretty minor difference here, but I'd prefer the use of a yaml file with the same fileIO setup I used for preferences. That is consistent with the rest of the application and gets us things like migrations if we ever need them. You should just be able to stick a list in one of them.

Reply:
Another thing: concatenating file paths like that isn't always good practice because different operating systems might have different schemes for it. The safest way is with `os.path.join`. I know I used the `+` for this in the application myself, but I have learned since then!